dask/dataframe/tests/test_arithmetics_reduction.py

```diff
@@ -729,13 +730,33 @@ def test_reductions(split_every):
             bias_factor = (n * (n - 1)) ** 0.5 / (n - 2)
             assert_eq(dds.skew(), pds.skew() / bias_factor)
 
+            if PANDAS_GT_200:
+                # TODO: Remove this `if`-block once `axis=None` support is added
```

Comment:
Might be helpful to open an issue and mention the issue here, so the person implementing it could grep the codebase for issue number.

Reply:
👍